frontend/src/Commons/Forms/InputModules/FieldsAndSoil/FieldsListComponent.tsx

```diff
@@ -87,7 +87,15 @@ const FieldsListComponent: FC<FieldListProps> = ({ farmDetails }) => {
                 marginRight="50px"
               >
                 <h2>Sampling Month</h2>
-                <p>{HasSoilTest ? SoilTest.sampleDate : ComponentText.NA}</p>
+                <p>
+                  {HasSoilTest
+                    ? new Date(SoilTest.sampleDate).toLocaleDateString('en-US', {
+                        timeZone: 'UTC',
+                        month: 'long',
+                        day: 'numeric',
```

Comment:
Could you change this to display month and year so it's the same as nmp? The day really doesn't matter much for the soil tests, it's mainly the month and then the year.

Reply:
Sure that's easy. I thought nmp was month and day as I've saw on soil test on nmp